opsmngr/opsmngr.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 const (
-	defaultBaseURL = atlas.CloudURL
```

Comment:
`atlas.CloudURL` is not longer available

Reply:
Should I add again `atlas.CloudURL` in the atlas go client? @gssbzn I didn’t notice that it was public when I removed it

Reply:
yes, please add it again

Reply:
https://github.com/mongodb/go-client-mongodb-atlas/pull/214